src/server/pkg/log/log.go

```diff
@@ -31,37 +31,52 @@ type Logger interface {
 
 type logger struct {
 	*logrus.Entry
-	histogram map[string]*prometheus.HistogramVec
-	counter   map[string]prometheus.Counter
-	mutex     *sync.Mutex
+	histogram   map[string]*prometheus.HistogramVec
+	counter     map[string]prometheus.Counter
+	mutex       *sync.Mutex
```

Comment:
This mutex is protecting `historgram` and `counter` right? If so a comment about that would be a nice addition. When a mutex is protecting one thing I generally like to do `counterMu` at which point a comment is redundant. But if it's protecting 2 `histogramCounterMu` is too dorky, so I prefer a comment.